ci-operator/config/openshift-priv/cluster-etcd-operator/openshift-priv-cluster-etcd-operator-release-4.20.yaml

```diff
@@ -234,6 +238,8 @@ tests:
   optional: true
   steps:
     cluster_profile: gcp
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest
```

Comment:
this is not an assisted job?

Reply:
I'm not sure about this change, we force all dependent jobs to change because we do something different. I think it can lead to issues because external teams don't have our internal knowledge, and will get failed jobs because they forgot to set this variable. I think the usage of our workflows should be as smooth as possible for others to re-use it, otherwise it may be a burden for us to support them.

Reply:
I understand, but we still want to install OCP relealse images from released streams, I can add this behaviour for all the external jobs as well, but it might still cause issues. we manage this jobs anyway, so I don't see the issue with providing it with valid config across all files - this is the same thing we have now, if you want to use `assisted-baremetal` workflow, you need to import all the required image-streams, the release, etc. Now you will need to choose the desired OCP release image stream as well, I don't think it is a major difference